pytest/assets/generic_file.txt

```diff
@@ -0,0 +1 @@
+HydroShare is great!
```

Comment:
It is great, but this file doesn't belong here :)

Reply:
I think it's ok here or if we want to be super strict, could move it into a newly created folder for whatever Django app this is centered on... so far I have pytest/functional/hs_core and functional/hs_discover so assets for these tests can be co-located in their final destination when Pabitra sorts that out, as long as it's not all in the top-level pytest/functional/